src/services/doppler-billing-api-client.ts

```diff
@@ -63,6 +64,15 @@ export class HttpDopplerBillingApiClient implements DopplerBillingApiClient {
     };
   }
 
+  private getDownloadUrl(links: any[]): string {
+    if (links) {
+      const link = links.find((l) => l.rel === 'file');
```

Comment:
Cuidado, el rel es un string que puede tener muchos términos, creo que en lugar de `===` deberías usar una expresión regular que acepte la palabra completa `file` incluso si hubiera más palabras, creo que deberías hacer tests para verificar estos casos: 
* `file otras palabras` → _Es un archivo_
* `otras palabras file` → _Es un archivo_
* `otras file palabras` → _Es un archivo_
* `fileotro` → _No es un archivo_
* `otrofile` → _No es un archivo_
* `file`

Reply:
perfect, I will work on it.

Reply:
Done.

**Changes:**

- Added the function: searchLinkByRegularExpressionRel in utils.ts. The function receives two parameters: the links and the regular expresion to search.
- Added unit tests for the scenarios above.